.github/workflows/node.js.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will run tests across different versions of node and different operating systems
+
+name: CI
+
+on:
+  pull_request:
+    branches: [ staging ]
+
+jobs:
+  test:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+        os: [windows-latest, macos-latest, ubuntu-latest]
```

Comment:
I don't think we need windows and macos oses, this can just cost us extra minutes (we have just 1000 free minutes)

Reply:
I didn't know that free is limited